### PR TITLE
Fix admonitions

### LINKED ISSUE #4452: Check admonitions
The following admonitions have to be checked:

- [ ] https://docs.iota.org/references/iota-api
- [ ] https://docs.iota.org/references/ts-sdk/typescript/

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Updated the admonitions mentioned by Thibault after confirming it with the related teams

## Links to any relevant issues

Fixes #4452.

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by lucas-tortora — APPROVED** [review]

File: docs/content/references/ts-sdk/typescript/index.mdx [hunk 1]
```diff
@@ -16,6 +17,9 @@ npm i @iota/iota-sdk
 
 The following table lists the locations for IOTA networks.
 
+{ /* TODO: Shhould be moved to mainnet once it is live */ }
```
Comment (Alex6323) [inline docs/content/references/ts-sdk/typescript/index.mdx]: Maybe create an issue for this TODO (if there isn't already) and put a link to it instead.

also, typo :stuck_out_tongue_winking_eye:

File: docs/content/_snippets/info-high-traffic.mdx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+:::info
+
+It is suggested to use dedicated nodes/shared services rather than public endpoints for production apps with high traffic volume.
+
+You can either run your own Full nodes, or outsource this to a professional infrastructure provider (preferred for apps that have high traffic).
```
Comment (thibault-martinez) [inline docs/content/_snippets/info-high-traffic.mdx:5]: Do we consistently uppercase `Full` in the docs? This always felt weird to me.

**Review by thibault-martinez — APPROVED** [review]

File: docs/content/_snippets/info-high-traffic.mdx (hunk 2 above)
Comment (Dr-Electron) [reply docs/content/_snippets/info-high-traffic.mdx:5]: I agree. Will change it

File: docs/content/references/ts-sdk/typescript/index.mdx (hunk 1 above)
Comment (Dr-Electron) [reply docs/content/references/ts-sdk/typescript/index.mdx]:
```suggestion
{ /* TODO: https://github.com/iotaledger/iota/issues/4497 */ }
```

File: docs/content/_snippets/info-high-traffic.mdx (hunk 2 above)
Comment (Dr-Electron) [reply docs/content/_snippets/info-high-traffic.mdx:5]: https://github.com/iotaledger/iota/issues/4498

**Pull request merged**